### PR TITLE
[Feature] Using account money view from card drawer

### PR DESCRIPTION
**Pull request by guchito9**
* Se parametriza account money, ahora la config de los colores y eso viene de backend.
* Pasamos a usar el layout que ahora está del lado de card drawer.

### CODE REVIEW

**Review by jorGonzalez291292 — APPROVED** [review]

File: example/src/main/java/com/mercadopago/android/px/utils/OneTapSamples.java [hunk 1]
```diff
@@ -188,7 +188,8 @@ public static void addAll(final Collection<Pair<String, MercadoPagoCheckout.Buil
     private static MercadoPagoCheckout.Builder startOneTapWithAccountMoneyAndCardsDebitCredit() {
 
         final IPaymentDescriptor payment = getGenericPaymentApproved();
-        final SplitPaymentProcessor samplePaymentProcessor = new SamplePaymentProcessor(payment);
+        final SplitPaymentProcessor samplePaymentProcessor = new SamplePaymentProcessor(
```
Comment (guchito9) [inline example/src/main/java/com/mercadopago/android/px/utils/OneTapSamples.java:191]: Este archivo hay que restaurarlo

File: gradle.properties [hunk 2]
```diff
@@ -30,7 +30,7 @@ okhttp=3.12.10
 gson=2.8.5
 retrofit=2.6.4
 # Other PX libs
-cardDrawer=2.+
+cardDrawer=LOCAL-2.10.0-2
```
Comment (guchito9) [inline gradle.properties:33]: Esto voy a restaurarlo

File: px-checkout/build.gradle [hunk 3]
```diff
@@ -53,7 +53,9 @@ dependencies {
 
     implementation "com.google.code.gson:gson:$gson"
     implementation "com.mercadolibre.android:ui:$ui"
-    implementation "com.mercadolibre.android:carddrawer:$cardDrawer"
+    implementation("com.mercadolibre.android:carddrawer:$cardDrawer") {
```
Comment (guchito9) [inline px-checkout/build.gradle:56]: Esto voy a restaurarlo

File: px-services/src/main/java/com/mercadopago/android/px/internal/services/CheckoutService.java [hunk 4]
```diff
@@ -20,7 +20,7 @@
     String CHECKOUT_VERSION = "v2";
     String ENVIRONMENT = BuildConfig.API_ENVIRONMENT_NEW;
 
-    @POST(ENVIRONMENT + "/px_mobile/" + CHECKOUT_VERSION + "/checkout")
+    @POST("https://run.mocky.io/v3/7188ceee-ed84-49e9-b303-a65d58ec3acb/" + ENVIRONMENT + "/px_mobile/" + CHECKOUT_VERSION + "/checkout")
```
Comment (guchito9) [inline px-services/src/main/java/com/mercadopago/android/px/internal/services/CheckoutService.java:23]: Esto voy a restaurarlo

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/slider/AccountMoneyFragment.kt [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    override fun getAccessibilityContentDescription(): String {
+        return model.description
+    }
+
+    companion object {
+        @JvmStatic fun getInstance(model: AccountMoneyDrawableFragmentItem): Fragment {
+            val instance = AccountMoneyFragment()
+            instance.storeModel(model)
```
Comment (jorGonzalez291292) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/slider/AccountMoneyFragment.kt:40]: acá podemos usar el `instance.also.{ }`

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/slider/AccountMoneyLowResFragment.kt [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+import com.mercadopago.android.px.internal.viewmodel.drawables.AccountMoneyDrawableFragmentItem
+
+internal class AccountMoneyLowResFragment : AccountMoneyFragment() {
+
+    override fun getLayout() = R.layout.px_fragment_account_money_low_res
+
+    companion object {
+        @JvmStatic fun getInstance(model: AccountMoneyDrawableFragmentItem): Fragment {
+            val instance = AccountMoneyLowResFragment()
+            instance.storeModel(model)
```
Comment (jorGonzalez291292) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/slider/AccountMoneyLowResFragment.kt:14]: idem anterior

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/drawables/AccountMoneyDrawableFragmentItem.kt [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+import androidx.fragment.app.Fragment
+import com.mercadopago.android.px.internal.util.KParcelable
+import com.mercadopago.android.px.internal.util.parcelableCreator
+import com.mercadopago.android.px.internal.viewmodel.CardUiConfiguration
+
+internal class AccountMoneyDrawableFragmentItem : DrawableFragmentItem, KParcelable {
+
+    val card: CardUiConfiguration
+
+    constructor(parameters: Parameters, card: CardUiConfiguration) : super(parameters) {
```
Comment (jorGonzalez291292) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/drawables/AccountMoneyDrawableFragmentItem.kt:13]: No lo podemos mover arriba en la clase?

**Comment by guchito9** [comment]
419K	./px-services/build/outputs/aar/services-release.aar
510K	./px-services/build/outputs/aar/services-debugBeta.aar
509K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
52K	./testlib/build/outputs/aar/testlib-release.aar
152K	./px-testlib/build/outputs/aar/testlib-debug.aar
152K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
136K	./px-testlib/build/outputs/aar/testlib-release.aar
3.6M	./px-checkout/build/outputs/aar/checkout-release.aar
4.0M	./px-checkout/build/outputs/aar/checkout-debug.aar
4.0M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Comment by guchito9** [comment]
416K	./px-services/build/outputs/aar/services-release.aar
505K	./px-services/build/outputs/aar/services-debugBeta.aar
505K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
52K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
122K	./px-testlib/build/outputs/aar/testlib-release.aar
1.7M	./px-checkout/build/outputs/aar/checkout-release.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debug.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Comment by guchito9** [comment]
416K	./px-services/build/outputs/aar/services-release.aar
505K	./px-services/build/outputs/aar/services-debugBeta.aar
505K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
51K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
122K	./px-testlib/build/outputs/aar/testlib-release.aar
1.7M	./px-checkout/build/outputs/aar/checkout-release.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debug.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Comment by guchito9** [comment]
416K	./px-services/build/outputs/aar/services-release.aar
505K	./px-services/build/outputs/aar/services-debugBeta.aar
505K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
51K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
122K	./px-testlib/build/outputs/aar/testlib-release.aar
1.7M	./px-checkout/build/outputs/aar/checkout-release.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debug.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Comment by guchito9** [comment]
416K	./px-services/build/outputs/aar/services-release.aar
505K	./px-services/build/outputs/aar/services-debugBeta.aar
505K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
51K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
122K	./px-testlib/build/outputs/aar/testlib-release.aar
1.7M	./px-checkout/build/outputs/aar/checkout-release.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debug.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Comment by guchito9** [comment]
416K	./px-services/build/outputs/aar/services-release.aar
505K	./px-services/build/outputs/aar/services-debugBeta.aar
505K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
51K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
122K	./px-testlib/build/outputs/aar/testlib-release.aar
1.7M	./px-checkout/build/outputs/aar/checkout-release.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debug.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Comment by guchito9** [comment]
416K	./px-services/build/outputs/aar/services-release.aar
506K	./px-services/build/outputs/aar/services-debugBeta.aar
505K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
51K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
122K	./px-testlib/build/outputs/aar/testlib-release.aar
1.7M	./px-checkout/build/outputs/aar/checkout-release.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debug.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/generic_modal/Actionable.kt [hunk 8]
```diff
@@ -1,34 +1,17 @@
...
 
+@Parcelize
 data class Actionable(
     val label: String,
     val deepLink: String?,
-    @ActionType val action: String?) : KParcelable {
+    @ActionType val action: String?) : Parcelable {
 
     init {
         if (deepLink.isNullOrEmpty() && action.isNullOrEmpty()) {
```
Comment (jorGonzalez291292) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/generic_modal/Actionable.kt:13]: Esto lo podemos cambiar por un `check(){}`?

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/drawables/PaymentMethodDrawableItemMapper.kt [hunk 9]
```diff
@@ -1,53 +1,76 @@
...
+            }
+        }
+        return initResponse?.let { initResponse ->
+            val genericDialogItem = expressMetadata.getBehaviour(CheckoutBehaviour.Type.TAP_CARD)?.modal?.let { modal ->
+                initResponse.modals[modal]?.let {
+                    FromModalToGenericDialogItem(ActionType.DISMISS, modal).map(it)
+                }
+            }
+            val parameters = getParameters(expressMetadata, initResponse.customSearchItems, genericDialogItem)
+            val cardUiMapper = CardUiMapper
```
Comment (jorGonzalez291292) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/drawables/PaymentMethodDrawableItemMapper.kt:42]: Hace falta hacer esto? `CardUiMapper` no es un `object`? En todo caso lo podemos inyectar.

File: px-services/src/main/java/com/mercadopago/android/px/internal/services/CheckoutService.java [hunk 10]
```diff
@@ -20,7 +20,7 @@
     String CHECKOUT_VERSION = "v2";
     String ENVIRONMENT = BuildConfig.API_ENVIRONMENT_NEW;
 
-    @POST(ENVIRONMENT + "/px_mobile/" + CHECKOUT_VERSION + "/checkout")
+    @POST("https://run.mocky.io/v3/f2f3e82a-d2d2-43b1-a767-24f8b9cbe935/" + ENVIRONMENT + "/px_mobile/" + CHECKOUT_VERSION + "/checkout")
```
Comment (jorGonzalez291292) [inline px-services/src/main/java/com/mercadopago/android/px/internal/services/CheckoutService.java:23]: Esto lo vamos a sacar antes de mergearlo?

Comment (guchito9) [reply px-services/src/main/java/com/mercadopago/android/px/internal/services/CheckoutService.java:23]: Si :P

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/drawables/PaymentMethodDrawableItemMapper.kt [hunk 11]
```diff
@@ -1,53 +1,76 @@
...
+            return when {
+                expressMetadata.isCard -> SavedCardDrawableFragmentItem(parameters, expressMetadata.paymentMethodId,
+                    cardUiMapper.map(expressMetadata.card.displayInfo))
+                expressMetadata.isAccountMoney -> expressMetadata.accountMoney.displayInfo.type?.let {
```
Comment (cgaggino) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/viewmodel/drawables/PaymentMethodDrawableItemMapper.kt:46]: creo que un takeif va a quedar mas cheto acá

**Review by cgaggino — APPROVED** [review]

**Comment by guchito9** [comment]
420K	./px-services/build/outputs/aar/services-release.aar
510K	./px-services/build/outputs/aar/services-debugBeta.aar
510K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
52K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
123K	./px-testlib/build/outputs/aar/testlib-release.aar
1.7M	./px-checkout/build/outputs/aar/checkout-release.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debug.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Comment by guchito9** [comment]
420K	./px-services/build/outputs/aar/services-release.aar
510K	./px-services/build/outputs/aar/services-debugBeta.aar
510K	./px-services/build/outputs/aar/services-debug.aar
57K	./testlib/build/outputs/aar/testlib-debug.aar
57K	./testlib/build/outputs/aar/testlib-debugBeta.aar
52K	./testlib/build/outputs/aar/testlib-release.aar
138K	./px-testlib/build/outputs/aar/testlib-debug.aar
138K	./px-testlib/build/outputs/aar/testlib-debugBeta.aar
123K	./px-testlib/build/outputs/aar/testlib-release.aar
1.7M	./px-checkout/build/outputs/aar/checkout-release.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debug.aar
1.9M	./px-checkout/build/outputs/aar/checkout-debugBeta.aar
61K	./px-addons/build/outputs/aar/addons-debug.aar
52K	./px-addons/build/outputs/aar/addons-release.aar
61K	./px-addons/build/outputs/aar/addons-debugBeta.aar

**Review by jorGonzalez291292 — APPROVED** [review]

**Pull request merged**